src/SensioLabs/Melody/Runner/Runner.php

```diff
@@ -27,10 +27,11 @@ public function __construct($vendorDir = 'vendor')
 
     public function getProcess(Script $script, $dir)
     {
-        if ($script->getResource() instanceof LocalResource) {
-            $bootstrap = $this->getLocalBootstrap($script->getResource());
+        $resource = $script->getResource();
+        if ($resource instanceof LocalResource && !$resource->hadShebang()) {
```

Comment:
Looking at code, I wonder if we shouldn't just always use the `getRemoteBootstrap`, that would avoid the `hasShebang` method

Reply:
I agree with @jderusse ; We could simplify the code here